links/src/utils/getParent.ts

```diff
@@ -0,0 +1,26 @@
+import { ToTree, Tree, GetSiblings } from "./generateLinks";
+
+// getParents(tree, '/cancerqld/research') get parent_id of slug
+export const getParents: GetSiblings = (arr, slug) => {
+	let tree: Tree = [];
+	for (const node of arr) {
+		if (node.slug !== slug) continue;
+		tree = closestParent(arr, node.parent_id);
+		tree.push(node);
+	}
+	return tree;
+};
+
+//closesrparent - get the second closest parent_id
```

Comment:
Make sure your comments are verbose and relevant [How to comment using JSDoc notation](https://devhints.io/jsdoc)